server/types/responses/eventResponse.ts

```diff
@@ -3,7 +3,6 @@ import { ObjectId } from "mongoose";
 export interface EventResponse {
 	_id: ObjectId;
 	name: string;
-	userid: ObjectId;
```

Comment:
Userids are no longer needed for the parent interface. `NationalEvents` do not require to be associated with a single user